tee-worker/ts-tests/parachain-api/build/interfaces/identity/definitions.ts

```diff
@@ -55,7 +55,7 @@ export default {
                     "(LitentryIdentity, LitentryIdentity, UserShieldingKeyType, H256)",
                 link_identity:
                     "(LitentryIdentity, LitentryIdentity, LitentryIdentity, LitentryValidationData, Vec<Web3Network>, UserShieldingKeyNonceType, H256)",
-                remove_identity: "(LitentryIdentity, LitentryIdentity, LitentryIdentity, H256)",
+                deactivate_identity: "(LitentryIdentity, LitentryIdentity, LitentryIdentity, H256)",
```

Comment:
why here only `deactivate_identity`, no `activate_identity`?

Reply:
no reason, just missed that 👍🏼